resources/css/typography.css

```diff
@@ -31,3 +31,8 @@ pre, code, .code {
   word-break: break-word;
   white-space: pre-wrap;
 }
+
+.text-overflow-wrap{
```

Comment:
Based on the rest of the file, I believe a space should be present between `wrap` and `{`.

Reply:
You are right, it was a small oversight from my side 😅 
Now it's fixed.